python/ray/experimental/serve/api.py

```diff
@@ -40,7 +47,9 @@ def init(kv_store_connector=None,
          http_host=DEFAULT_HTTP_HOST,
          http_port=DEFAULT_HTTP_PORT,
          ray_init_kwargs={"object_store_memory": int(1e8)},
-         gc_window_seconds=3600):
+         gc_window_seconds=3600,
+         queueing_policy="random",
```

Comment:
I think you can just use enum here? `Policy.Random`

Reply:
exposing Policy.random would expose the queueing policy code to the user! Which I thought should be kept only for the implementer

Reply:
How about queuing_policy: Policy enum, or CentralizedQueues actor subclass.

Reply:
Are you okay with the user importing Policy Enum class?

Reply:
Well now i think about it, user don't need to explicitly import policy enum if we add policy to serve namespace.
```python
from ray.experimental import serve
serve.init(queuing_policy=serve.Policy.RoundRobin)
```

Reply:
Okay!